File: labs/solidworks/swex/add-in/commands-manager/defining-commands/CommandsManager.CommandsScope.cs
using CodeStack.SwEx.AddIn.Attributes;
using CodeStack.SwEx.AddIn.Enums;

public enum CommandsD_e
{
    [CommandItemInfo(swWorkspaceTypes_e.Part)]
    CommandD1,

    [CommandItemInfo(swWorkspaceTypes_e.Part | swWorkspaceTypes_e.Assembly)]
    CommandD2
}

File: labs/solidworks/swex/add-in/commands-manager/defining-commands/CommandsManager.CommandsScope.vb
Imports CodeStack.SwEx.AddIn.Attributes
Imports CodeStack.SwEx.AddIn.Enums

Public Enum CommandsD_e

    <CommandItemInfo(swWorkspaceTypes_e.Part)>
    CommandD1

    <CommandItemInfo(swWorkspaceTypes_e.Part Or swWorkspaceTypes_e.Assembly)>
    CommandD2

End Enum
